x-pack/docs/en/rest-api/security/create-api-keys.asciidoc

```diff
@@ -67,6 +65,51 @@ role descriptor with no privileges. The derived API key can be used for
 authentication; it will not have authority to call {es} APIs.
 
 --
++
```

Comment:
I've tried defining a file with a common role descriptor definition and then including it in both `create-roles.asciidoc` and `create-api-keys.asciidoc`, but I had issues with indentation and conditionals. So I decided to simply copy the whole definition here. Not ideal, but I couldn't make it work otherwise.

Reply:
Right, the indentation level is different. There might be ways around it. But since it's only copied once, I am also OK with it.